### ISSUE TITLE
Uncaught Error: Could not find module htmlbars-runtime

### ISSUE THREAD

**Issue by Iftahh** [ember-animation/liquid-fire#555]
Trying to update my project to latest version of dependencies I hit a runtime exception when the app loads - see stack trace below.

Google found a similar error in https://github.com/ef4/liquid-fire/issues/287  but seems its too old to be relevant.

When I remove liquid-fire the exception disappears.

I'm using:
  liquid-fire@0.27.0
  ember-source@2.11.2
  ember-cli: 2.11.1

```
    Uncaught Error: Could not find module htmlbars-runtime
    at missingModule (ember.debug.js:58)
    at internalRequire (ember.debug.js:80)
    at requireModule (ember.debug.js:44)
    at Module.callback (ember-internals.js:24)
    at Module.exports (loader.js:131)
    at Module._reify (loader.js:164)
    at Module.reify (loader.js:150)
    at Module.exports (loader.js:129)
    at requireModule (loader.js:23)
    at resolveInitializer (index.js:5)
    at registerInitializers (index.js:18)
    at exports.default (index.js:44)
    at Module.callback (app.js:13)
    at Module.exports (loader.js:131)
    at requireModule (loader.js:23)
    at app-boot.js:4
```

**Comment by Iftahh** [comment]
A brand new app with the same setup works just fine, so there must be something else in my app that triggers the error, but I don't know what it is or how to find debug it.

> Is there any more data I can provide to make it clearer?

**Comment by ivanvanderbyl** [comment]
This is probably because of `ember-source` not being detected (since switching from bower to npm), it's trying to include itself as though your app is pre Ember 2.x. I'll open a PR with a patch soon.

**Comment by ivanvanderbyl** [comment]
There seems to be an issue with ember version specific loading, and for some reason you're getting the version for Ember 1.x instead 2.x. I'm experiencing the same issue, and now looking into the initialization to figure out why. I'll open a PR if I figure it out.

**Comment by mekeor** [comment]
Could you please bump the version number of the liquid-fire package? This bug affects me but I can't easily benefit from the solution of the bug as long as there's no new version. 😉  Thanks ❤️

**Comment by Duder-onomy** [comment]
This is now a blocking issue in Ember **2.12**

**Comment by kylemellander** [comment]
I am experiencing the block in Ember 2.12

**Comment by kylemellander** [comment]
After researching this, it seems like the fix is merged into master, but it is not published.  Do we have a timetable on when this will be pushed since it is a breaking bug?

**Comment by alexspeller** [comment]
This issue seems to be resolved by `0.27.2`, I think it can be closed now
